manifests/agent.pp

```diff
@@ -218,8 +216,10 @@
           fail('Opsmatic Agent is not supported on this platform')
         }
       }
+
     }
     default: {
+      include opsmatic::monitor_files
```

Comment:
why do we include monitor_files in the block which appears to uninstall the agent?